content/zh-cn/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -18,60 +30,78 @@
 
 		<div class="row">
 			<div class="col-md-8">
-<!--        <h3>Objectives</h3>-->
+				<!-- <h3>Objectives</h3>-->
```

Comment:
```suggestion
			<!-- <h3>Objectives</h3>-->
```
A localized version needs to follow the en upstream including text, layout, indentations, and more.